src/Components/VerticalLineDivide.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function VerticalLineDivide() {
+  return (
+    <div>
+      <div className="border-solid border-l-2 border-blue-500 h-20 ml-2"></div>
+    </div>
```

Comment:
why you create 2 div?
it should be better if you write like this
 return (
      <div className="border-solid border-l-2 border-blue-500 h-20 ml-2"></div>
    )
do the same for other components